src/NodeBuilder.js

```diff
@@ -208,26 +209,14 @@ class NodeJsBuilder {
       this.nodePath('vcbuild.bat'),
       join(this.patchDir, 'vcbuild.bat.patch'));
 
-    await patchFile(
```

Comment:
I suggest reverting your changes from lines 211-222. Let's maintain backward compatibility in case someone wants to regenerate the patches for versions like 18.15.0, where the patches are included. Handling missing patches is already covered in the patchfile implementation.